mostlyai/generators.py

```diff
@@ -75,7 +75,44 @@ def create(self, config: dict) -> Generator:
         )
         return generator
 
+    def import_from_file(
```

Comment:
I guess it's coming up in the next commit: `_import_from_file` in `model.py:Generator` (same in `tools/...`)

Reply:
import_from_file method is not going to be executed on a specific generator, but one level higher